### PR TITLE
White label logic

### PR DESCRIPTION
**Pull request by CalebPena**
What (if any) features are you implementing?
- Filter programs based on the screen white label.
- Filter urgent needs based on the screen white label.
- Filter web hooks based on the screen while label.
- Pick the CMS integration based on the screen white label.
- Add white label select when adding something in the translation admin.
- Add `stepDirectory` to referrerData in the config.

### Review and merge https://github.com/Gary-Community-Ventures/benefits-api/pull/646 first

### CODE REVIEW

**Comment by CalebPena** [comment]
@kbrey Please take a look at the changes to the config, so y'all can implement them on your end.

**Review by ivonne-hernandez — APPROVED** [review]

File: configuration/admin.py [hunk 1]
```diff
@@ -12,7 +12,14 @@ class ConfigurationAdmin(ModelAdmin):
             "widget": JSONEditorWidget(options={"modes": ["tree", "code"], "mode": "tree", "enableDrag": False})
         },
     }
-    search_fields = ("name",)
+    search_fields = ("name", "white_label__name")
```
Comment (kbrey) [inline configuration/admin.py:15]: Does "white_label__name" have an extra underscore here?

File: configuration/admin.py [hunk 2]
```diff
@@ -12,7 +12,14 @@ class ConfigurationAdmin(ModelAdmin):
...
+    list_display = ("name", "white_label_name")
+
+    def white_label_name(self, obj):
+        return obj.white_label.name
+
+    white_label_name.admin_order_field = "white_label__name"
```
Comment (kbrey) [inline configuration/admin.py:21]: Same question here, is there an extra underscore?

File: configuration/white_labels/nc.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from .base import ConfigurationData
+from screener.models import WhiteLabel
+
+
+# TODO: Update NC configuration
```
Comment (kbrey) [inline configuration/white_labels/nc.py:5]: Is this something our team will be responsible for?

File: configuration/admin.py (hunk 1 above)
Comment (CalebPena) [reply configuration/admin.py:15]: Yes, the syntax in Django for filtering by a nested field is to use a double underscore. This is saying to search with the `name` field on the `white_label` model.

**Pull request merged**